src/SharpCompress/Readers/IReader.cs

```diff
@@ -8,6 +8,7 @@ public interface IReader : IDisposable
     {
         event EventHandler<ReaderExtractionEventArgs<IEntry>> EntryExtractionBegin;
```

Comment:
I sort of agree, but this would break current implementations using those event handlers. Would this be ok?